frontend/sozisel-app/src/apolloClient.ts

```diff
@@ -24,7 +25,10 @@ const createAbsintheUploadLink = require("apollo-absinthe-upload-link")
 const HOST = window.location.hostname;
 
 function createApolloHttpLink(): ApolloLink {
-  return createAbsintheUploadLink({ uri: `http://${HOST}:4000/api` });
+  return createAbsintheUploadLink({
+    uri: `http://${HOST}:4000/api`,
```

Comment:
You can take protocol from `window.location` as well, just like you do this couple of times in this PR